packages/shared/src/components/buttons/Button.tsx

```diff
@@ -46,6 +46,7 @@ export interface BaseButtonProps {
   displayClass?: string;
   textPosition?: string;
   position?: string;
+  disabled?: boolean;
```

Comment:
Since `HTMLAnchorElement` does not have `disabled` attribute I added `disabled` in `BaseButtonProps` to allow for any button to pass in disabled state as we do support it in `Tag`.

This does not show when calling `<Button tag="a" disabled />` for some reasons, probably due some kind of TS misconfiguration.

It did show if you did the following:
```tsx
const props = {
  tag: 'a',
  disabled: false
}

<Button {...props} />
```

All in all now it's fixed.